spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb

```diff
@@ -120,6 +122,42 @@
     end
   end
 
+  describe '#partner_agency' do
+    subject(:partner_agency) { presenter.partner_agency }
+
+    it 'returns partner agency' do
+      expect(partner_agency).to eq('http://localhost:3000')
+    end
+  end
+
+  describe '#days_reminder' do
+    subject(:days_reminder) { presenter.days_reminder }
+ 
+    context '4 days until due date' do
+      it 'returns 3 days' do
+        travel_to(enrollment_established_at + 26.days) do
```

Comment:
Since the expiration timeframe is configurable, can we base the offset here from the configuration value, so that the tests don't start failing if we changed the configuration?

```suggestion
        travel_to(enrollment_established_at + (IdentityConfig.store.in_person_enrollment_validity_in_days - 4).days) do
```

Or at least stub the configuration value so that we have stronger guarantees of the timeframe we're testing against?

```rb
before do
  allow(IdentityConfig.store).to receive(:in_person_enrollment_validity_in_days).and_return(30)
end
```